src/pseudopeople/noise_functions.py

```diff
@@ -681,34 +692,35 @@ def _corrupt_tokens(
             )
             if eligible.sum() == 0:
                 continue
+            del long_enough
```

Comment:
Is this for memory concerns?

Reply:
Yes; I don't know how much it matters but it seems wasteful to keep many arrays that are as long as our data when we don't need to